WooCommerce/Classes/ViewRelated/Orders/OrderDetailsViewController.swift

```diff
@@ -36,6 +36,16 @@ class OrderDetailsViewController: UIViewController {
         return EntityListener(storageManager: AppDelegate.shared.storageManager, readOnlyEntity: viewModel.order)
     }()
 
+    /// ResultsController: Surrounds us. Binds the galaxy together. And also, keeps the UITableView <> (Stored) OrderStatuses in sync.
```

Comment:
@jleandroperez's `/// ResultsController: Surrounds us. Binds the galaxy together...` still lives on ❤️